gcp/appengine/frontend_handlers.py

```diff
@@ -344,6 +345,14 @@ def markdown(text):
   return ''
 
 
+@blueprint.app_template_filter('display_json')
+def display_json(data):
+  # We can't use the default `tojson` filter as it's intended for code (and
+  # escapes characters like '<' to '\u003c'). We want to render the JSON for
+  # display purposes and render '<' as-is using HTML escaping instead.
```

Comment:
nit: maybe add something like `(e.g. &lt;)` to this comment section here to make it clearer what's meant by HTML escaping, by skimming this section I thought it was literal `<`

Reply:
clarified!